opendevin/config.py

```diff
@@ -145,3 +146,8 @@ def get(key: str, required: bool = False):
     if not value and required:
         raise KeyError(f"Please set '{key}' in `config.toml` or `.env`.")
     return value
+
+
+_cache_dir = config.get('CACHE_DIR')
+if _cache_dir:
+    pathlib.Path(_cache_dir).mkdir(parents=True, exist_ok=True)
```

Comment:
I'm not entirely certain of the purpose of cache dir--do we still need this?

Reply:
We need this (at least for now) to cache the `pip install` commands in Jupyter requirement dependency - without this, you need to re-download the `whl` for *every* sandbox initialization, which can be way too slow. I still felt it is a bit slow with cache... we probably need to figure out a way to further cache it in the future (e.g., save the docker image with the requirement installed, and re-use it as much as possible)